src/main/java/trade/wayruha/whitebit/dto/SingleMarketActivity.java

```diff
@@ -0,0 +1,17 @@
+package trade.wayruha.whitebit.dto;
+
+import lombok.Data;
+
+@Data
+public class SingleMarketActivity {
```

Comment:
it's better to have "Single" class as a static inner class inside its context, i.e., MarketActivity in this case.